### PR TITLE
update readme - sensor simulator

### PR DESCRIPTION
**Pull request by Aquijada52**
line 32: device type name will need to be updated as well, added issue to project board

@upkarlidder @krook ready for review/merge

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
-# Prometeo sensor simulator
+# Pyrrha sensor simulator
 
-This repository will contain the [Prometeo](https://github.com/Call-for-Code/Prometeo) solution sensor simulator.
+This repository will contain the [Pyrrha](https://github.com/Call-for-Code/Prometeo) solution sensor simulator.
```
Comment (krook) [inline README.md]: Need to update the URL too

**Review by krook — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Aquijada52) [reply README.md]: https://github.com/Pyrrha-Platform/Pyrrha 
going to go with this one?

Comment (krook) [reply README.md]: Yes please

Comment (Aquijada52) [reply README.md]: url has been updated

**Review by krook — APPROVED** [review]

**Pull request merged**